cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfBotPawn.cpp

```diff
@@ -47,9 +47,31 @@ AUrdfBotPawn::AUrdfBotPawn(const FObjectInitializer& object_initializer) : APawn
 void AUrdfBotPawn::SetupPlayerInputComponent(class UInputComponent* input_component)
 {
     Super::SetupPlayerInputComponent(input_component);
+
+    UPlayerInput* player_input = GetWorld()->GetFirstPlayerController()->PlayerInput;
+    auto keyboard_actions = Config::get<std::map<std::string, std::map<std::string, std::map<std::string, float>>>>("URDFBOT.URDFBOT_PAWN.KEYBOARD_ACTIONS");
+    for (auto& keyboard_acton : keyboard_actions) {
+        InputAxisBinding input_axis_binding;
+        input_axis_binding.axis_ = FName(GetName() + "::" + keyboard_acton.first.c_str());
```

Comment:
input_axis_binding.axis_ = Unreal::toStdString(GetName()) + "::" + keyboard_action.first;

Reply:
I still like std::string because it's easier to examine in a debugger like XCode or Visual Studio.

Reply:
fixed.